compiler/rustc_middle/src/middle/region.rs

```diff
@@ -96,6 +96,7 @@ impl fmt::Debug for Scope {
             ScopeData::Arguments => write!(fmt, "Arguments({:?})", self.id),
             ScopeData::Destruction => write!(fmt, "Destruction({:?})", self.id),
             ScopeData::IfThen => write!(fmt, "IfThen({:?})", self.id),
+            ScopeData::IfThen2024 => write!(fmt, "IfThen[edition2024]({:?})", self.id),
```

Comment:
Remark [REV3,ED2024]: mentions of Edition 2024.

Reply:
Lemme keep it for now